Corona-Warn-App/src/main/res/drawable/ic_arrow_right.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
isn't this the same arrow that we use in the contact journal? `ic_contact_diary_right_arrow.xml` couldn't we reuse the asset?

Reply:
checked both proposed arrows but unfortunately none of those match the color...

Reply:
Tinting?

Reply:
Done -> ic_contact_diary_right_arrow.xml is now reused with alpha="0.6"

Reply:
Thanks 👍